### PR TITLE
Fix Windows build command highlighting.

### PR DESCRIPTION
**Pull request by bruvzg**
Fixes highlighting:

| Before | After |
| -- | -- |
| Screenshot 2023-12-21 at 21 34 20 | Screenshot 2023-12-21 at 21 34 31 |

### CODE REVIEW

File: contributing/development/compiling/compiling_for_windows.rst [hunk 1]
```diff
@@ -132,7 +132,9 @@ Running SCons
 ~~~~~~~~~~~~~
 
 After opening a command prompt, change to the root directory of
-the engine source code (using ``cd``) and type::
+the engine source code (using ``cd``) and type
```
Comment (AThousandShips) [inline contributing/development/compiling/compiling_for_windows.rst:135]:
```suggestion
the engine source code (using ``cd``) and type:
```
I'd keep the colon here, same everywhere (as it's printed as a single colon when using `::`)

File: contributing/development/compiling/introduction_to_the_buildsystem.rst [hunk 2]
```diff
@@ -93,7 +93,9 @@ generally with this naming convention::
 
     godot.<platform>.<target>[.dev][.double].<arch>[.<extra_suffix>][.<ext>]
 
-For the previous build attempt, the result would look like this::
+For the previous build attempt, the result would look like this
```
Comment (AThousandShips) [inline contributing/development/compiling/introduction_to_the_buildsystem.rst:96]:
```suggestion
For the previous build attempt, the result would look like this:
```
Ditto

**Review by AThousandShips — APPROVED** [review]

**Review by growingbrain — APPROVED** [review]
Thanks, looks much better.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Comment by YuriSizov** [comment]
Oh, didn't know about `doscon`. Although we lose highlighting for the scons arguments, and overall the prefix part is not really useful (I don't know why there is a tradition to show it, or like `$` for Linux shells), and can probably be simply removed from the command.

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2.

**Pull request merged**